app/presenters/wpcc/period_contribution_presenter.rb

```diff
@@ -32,10 +32,24 @@ def formatted_total_contributions
       formatted_contribution(object.total_contributions)
     end
 
+    def employer_frequency_heading(salary_frequency)
+      i18n_key = 'wpcc.results.period.contribution_heading.employers'
+
+      if fourweekly_welsh_translation?(salary_frequency)
+        t("#{i18n_key}_4_weekly", salary_frequency: salary_frequency.to_adj)
+      else
+        t(i18n_key, salary_frequency: salary_frequency.to_adj)
+      end
+    end
```

Comment:
This is good 👍 but reading again ... Wdyt on this method, something like:

```
default_i18n_key = 'wpcc.results.period.contribution_heading.employers'
adjective = salary_frequency.to_adj
specific_i18n_key = "#{i18n_key}_#{adjective}"

I18n.t(specific_i18n_key, default: :"#{default_i18n_key}", salary_frequency: adjective)
```

This will try to render the adjective first and in case there isn't one they will translate the default message.

You can see the I18n defaults [here](http://www.rubydoc.info/github/svenfuchs/i18n/I18n/Base#translate-instance_method)